src/ucp/api/ucp.h

```diff
@@ -1303,6 +1304,13 @@ typedef struct {
      */
     ucp_datatype_t datatype;
 
+    /**
+     * Memory type of the buffer. see @ref ucs_memory_type_t for possible memory types.
+     * If this value is not set (along with its corresponding bit in the op_attr_mask -
+     * @ref UCP_OP_ATTR_FIELD_MEMORY_TYPE), then use default @ref UCS_MEMORY_TYPE_UNKNOWN.
```

Comment:
` then use default @ref UCS_MEMORY_TYPE_UNKNOWN which means the memory type will be detected automatically`